src/bacnet/basic/object/csv.c

```diff
@@ -80,29 +78,70 @@ void CharacterString_Value_Property_Lists(
     return;
 }
 
+/**
+ * @brief Creates a CharacterString Value object
+ * @param object_instance - object-instance number of the object
+ * @return the object-instance that was created, or BACNET_MAX_INSTANCE
+ */
+uint32_t CharacterString_Value_Create(uint32_t object_instance)
```

Comment:
are there more _Create() functions coming for the other object types?

Reply:
There are... This one had not been yet updated by the upstream.